src/panels/config/entities/entity-registry-settings.ts

```diff
@@ -62,7 +62,36 @@ const OVERRIDE_DEVICE_CLASSES = {
     "shutter",
     "window",
   ],
-  binary_sensor: ["window", "door", "garage_door", "opening"],
+  binary_sensor: [
```

Comment:
Can we only show every possible option when you're in advanced mode?

Reply:
Yep. Need to set that up. Also need to figure out the categories